man/systemd-pstore.service.xml

```diff
@@ -81,6 +81,20 @@
       <citerefentry><refentrytitle>pstore.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
       </para>
     </refsect2>
+
+    <refsect2>
+      <title>Controlling kernel parameters</title>
+
+      <para> The kernel has two parameters, crash_kexec_post_notifiers and
+      printk.always_kmsg_dump, that control writes into pstore.
+      The crash_kexec_post_notifiers parameter enables the kernel to write
+      dmesg (including stack trace) into pstore upon a panic or crash, and
+      printk.always_kmsg_dump parameter enables the kernel to write dmesg
+      upon a shutdown (shutdown, reboot, halt). These kernel parameters are
+      managed in tmpfiles.
```

Comment:
maybe say which file exactly, i.e. `… are managed via the <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry> mechanism, specifically the file <filename>/usr/lib/tmpfiles/systemd-pstore.conf</filename>.`

Reply:
done